src/main/main.js

```diff
@@ -848,12 +852,15 @@ window.interpretInputs = function(i,active){
     }
   }
 
-  $("#lsAxisX"+i).empty().append(lstickX.toFixed(5));
-  $("#lsAxisY"+i).empty().append(lstickY.toFixed(5));
-  $("#csAxisX"+i).empty().append(cstickX.toFixed(5));
-  $("#csAxisY"+i).empty().append(cstickY.toFixed(5));
-  $("#lAnalog"+i).empty().append(lAnalog.toFixed(5));
-  $("#rAnalog"+i).empty().append(rAnalog.toFixed(5));
+  if (showDebug) {
+    $("#lsAxisX"+i).empty().append(lstickX.toFixed(5));
+    $("#lsAxisY"+i).empty().append(lstickY.toFixed(5));
+    $("#csAxisX"+i).empty().append(cstickX.toFixed(5));
+    $("#csAxisY"+i).empty().append(cstickY.toFixed(5));
+    $("#lAnalog"+i).empty().append(lAnalog.toFixed(5));
+    $("#rAnalog"+i).empty().append(rAnalog.toFixed(5));
+  }
```

Comment:
At some point I'll want to cache these too, but it was a bit harder since the IDs are dynamically calculated. But it's behind debug anyway.